Software/Python/easygopigo3.py

```diff
@@ -476,6 +476,21 @@ def reset_encoders(self):
         self.offset_motor_encoder(self.MOTOR_LEFT,self.get_motor_encoder(self.MOTOR_LEFT))
         self.offset_motor_encoder(self.MOTOR_RIGHT,self.get_motor_encoder(self.MOTOR_RIGHT))
 
+    def read_encoders(self):
```

Comment:
@CleoQc just want  you to be aware of this, is there something wrong with the encoder readings?